package.json

```diff
@@ -60,7 +65,7 @@
     "@sinonjs/formatio": "^3.2.1",
     "@sinonjs/text-encoding": "^0.7.1",
     "just-extend": "^4.0.2",
-    "lolex": "^4.1.0",
+    "lolex": "^5.0.1",
```

Comment:
Just a side note: should we make `lolex` a peerDependency? I mean, it's included as part of `nise` and `sinon`, and it means that it's usually included twice, as two different versions, bloating the package. Not that this means much for a testing lib, but it's a bit of a drag to have to update both `nise` and `sinon` every time Lolex changes just to keep them in sync.

Reply:
That's not entirely true. We have to update both for semver major releases only. Otherwise the version installed in Sinon wins. This is not changing with a `peerDependency`. We'd still have to update majors in both projects.

I thought `nise` is supposed to function as an independent package and therefore should ship with `lolex` included. That wouldn't be the case anymore if we made it a `peerDependency`.